### PR TITLE
added the user interface files

### PR DESCRIPTION
**Pull request by ombothe08**
noworning

### CODE REVIEW

File: CleaningBotPathGenerator/postcss.config.js
Comment (vinayaksutar) [inline CleaningBotPathGenerator/postcss.config.js:1]: Are these plugins needed?

File: CleaningBotPathGenerator/public/vite.svg
Comment (vinayaksutar) [inline CleaningBotPathGenerator/public/vite.svg:1]: Is this icon needed?

File: CleaningBotPathGenerator/src/App.css
Comment (vinayaksutar) [inline CleaningBotPathGenerator/src/App.css:1]: Clean this file. Remove unused css.

File: CleaningBotPathGenerator/src/Components/CanvasContainer.tsx
Comment (vinayaksutar) [inline CleaningBotPathGenerator/src/Components/CanvasContainer.tsx:1]: I think file name is not correct.

File: CleaningBotPathGenerator/src/helpers/CleaningBot.js
Comment (vinayaksutar) [inline CleaningBotPathGenerator/src/helpers/CleaningBot.js:1]: Why .js?

File: CleaningBotPathGenerator/src/index.css
Comment (vinayaksutar) [inline CleaningBotPathGenerator/src/index.css:1]: Need?

File: CleaningBotPathGenerator/src/vite-env.d.ts
Comment (vinayaksutar) [inline CleaningBotPathGenerator/src/vite-env.d.ts:1]: remove commented code

File: CleaningBotPathGenerator/tailwind.config.js
Comment (vinayaksutar) [inline CleaningBotPathGenerator/tailwind.config.js:1]: remove if not needed

File: CleaningBotPathGenerator/src/index.css
Comment (ombothe08) [reply CleaningBotPathGenerator/src/index.css:1]: To import the tailwind CSS it required mentioned in the documentation

File: CleaningBotPathGenerator/tailwind.config.js
Comment (ombothe08) [reply CleaningBotPathGenerator/tailwind.config.js:1]: it needed

File: CleaningBotPathGenerator/src/index.css
Comment (vinayaksutar) [reply CleaningBotPathGenerator/src/index.css:1]: It's their answer, have you checked from your side?

File: CleaningBotPathGenerator/tailwind.config.js
Comment (vinayaksutar) [reply CleaningBotPathGenerator/tailwind.config.js:1]: What is tailwind?

File: CleaningBotPathGenerator/src/index.css
Comment (ombothe08) [reply CleaningBotPathGenerator/src/index.css:1]: yes

File: CleaningBotPathGenerator/tailwind.config.js
Comment (ombothe08) [reply CleaningBotPathGenerator/tailwind.config.js:1]: CSS framework with predefined classes

**Review by vinayaksutar — APPROVED** [review]

**Pull request merged**